halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/plugins/AddPluginCommand.java

```diff
@@ -41,6 +43,9 @@
   @Parameter(names = "--enabled", description = "To enable or disable the plugin.")
   private String enabled;
 
+  @DynamicParameter(names = "-O", description = "Set custom options, must be key=value format.")
+  private HashMap<String, Object> options = new HashMap<>();
```

Comment:
How can it be an `Object`? How can users pass something else than a string? An object? A number? a date? etc.

Reply:
Also this has to be `Serializable` at least (not necessarily explicitly). Either change to `String` or show how you could pass that information (with tests)